go.mod

```diff
@@ -154,3 +170,5 @@ toolchain go1.21.0
 replace github.com/jsipprell/keyctl => github.com/hallyn/keyctl v1.0.4-0.20230720164202-b9476cd969e8
 
 replace github.com/containers/image/v5 => github.com/hallyn/image/v5 v5.0.0-20230329035821-bb0673fd450d
+
+replace github.com/project-machine/machine => github.com/hallyn/machine v0.0.0-20230928184440-db1ec7acdad6
```

Comment:
It requires an open PR.  Since that PR introduces a new API, I won't just merge it - that would be rude, as it sort of locks machine into promises which weren't agreed to.